README.md

```diff
@@ -1897,6 +1897,7 @@ In other words, the file structure on the disk looks something like:
                 myplugin.py
 
 yt-dlp looks for these `yt_dlp_plugins` namespace folders in many locations (see below) and loads in plugins from **all** of them.
+Set the environment variable `YTDLP_NO_PLUGINS` to something nonempty to disable loading plugins entirely.
```

Comment:
This should not be user-facing and should be folded into something like `--plugin-dir none` imo

Reply:
I agree, I didn't realise this was merged. It should not be documented..

 --no-plugins is being added in https://github.com/yt-dlp/yt-dlp/pull/11305

Reply:
I disagree. This is a valid usecase and `--no-plugins` was not yet there. It doesn't harm to have a variable like this, which is also namespaced as to not clobber other variables names.